draft-ietf-rtcweb-jsep.xml

```diff
@@ -1513,7 +1515,8 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               new ICE credentials, for the purpose of forcing an ICE
               restart and kicking off a new gathering phase, in which
               the new servers will be used. If the ICE candidate pool
-              has a nonzero size, any existing candidates will be
+              has a nonzero size, and a non-provisional local description
+              has not yet been applied, any existing candidates will be
```

Comment:
This doesn't seem like quite the right conditional, if I understand the code correctly. Consider

- set pool size = 5
- addTrack()
- createOffer()/setLocal(). We now have 1 set of candidates and a pool size of 4
- setConfiguration() (but we don't discard the pool)l